extensions/git/package.json

```diff
@@ -1361,6 +1361,11 @@
           "description": "%config.branchWhitespaceChar%",
           "default": "-"
         },
+        "git.commitMessageFontFamily": {
```

Comment:
Since this applies to any SCM, not just git, the setting needs to come from here instead:

https://github.com/microsoft/vscode/blob/ee8a6bc4f8a46cd709111b231d950b4468981676/src/vs/workbench/contrib/scm/browser/scm.contribution.ts#L76

Reply:
Also, `commit` is a git term, usually we call it `input` in SCM. So it would be something like `scm.inputFontFamily`.

Reply:
Moved this contribution to SCM.